services/monitoring/docker-compose.yml

```diff
@@ -18,7 +18,7 @@ networks:
 services:
 
   prometheus:
-    image: prom/prometheus
+    image: prom/prometheus:latest
```

Comment:
shouldn't we fix a version and then have a bot that creates a branch, upgrades to latest version each image and run the tests automatically?? 

We should check tools like https://renovatebot.com/

Reply:
created a separate issue for this. As I did not change any behavior here.
I agree we should try that tool.